src/OpenAL/Extensions/Silk.NET.OpenAL.Extensions.Creative/EffectExtension.cs

```diff
@@ -11,12 +11,12 @@ namespace Silk.NET.OpenAL.Extensions.Creative
     /// <summary>
     /// Exposes the functions of the Effects Extension.
     /// </summary>
-    [Extension("AL_EXT_EFX")]
+    [Extension("ALC_EXT_EFX")] // the ALC_ prefix is misleading
     [NativeApi(Prefix = "al")]
```

Comment:
You sure?

Reply:
Absolutely positive, it's frustrating and makes me sick to my stomach that this is correct but having looked over the official headers and based on feedback from our users this is correct.